tests/qos/qos_sai_base.py

```diff
@@ -7,6 +7,7 @@
 import random
 import os
 import sys
+import six
```

Comment:
There is already a open #7958

Reply:
Thanks for the information. I will try to fix conflict after #7958 is merged.